Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.17.6-bullseye
+
+ENV UID=1000
+ENV GID=1000
+ENV GOXYGEN_DOCKER=true
+
+WORKDIR /app
+COPY . .
+
+# Without this line, the docker image will not be able to run and produce error: 
+# `Failed to setup a Git repository: exit status 128`
+# Because there is a line of code that stage files to the git repository and commit on
+# codegen > codegen.go > function initGitRepo 
+RUN git config --global user.email "arlhba@gmail.com" && git config --global user.name "Luqmanul Hakim"
```

Comment:
I would suggest using a non-existing email for code generation. Eg: `goxygen@noreply.github.com` with name set as `Goxygen`.

If you want to preserve attribution in the image, you can add a label that would end up in the image metadata once it is pushed:
```
LABEL org.opencontainers.image.authors="arlhba@gmail.com"
```